tools/orc/main.go

```diff
@@ -531,6 +556,84 @@ func showTdxComponent(indent string, bnd *bundle.Bundle, comp *bundle.Component)
 	fmt.Printf("%s  Memory:  %d MiB\n", indent, comp.TDX.Resources.Memory)
 }
 
+// sgxGenerateKey generates a 3072-bit RSA key with public exponent 3 as required for SGX.
+//
+// The code below is adopted from the Go standard library as it is otherwise not possible to
+// customize the exponent.
+func sgxGenerateKey(random io.Reader) (*rsa.PrivateKey, error) {
```

Comment:
Should we add some kind of test, e.g. to test that `e == 3`, `bits == 3072`, two primes, ...?